tide/master/MasterConfiguration.h

```diff
@@ -108,6 +108,13 @@ class MasterConfiguration : public Configuration
     const QString& getWebBrowserDefaultURL() const;
 
     /**
+     * Get the URL used for saving whiteboard
+     * @return The URL defined in the configuration file, or a default value if
+     * none is found.
+     */
+    const QString& getWhiteboardDefaultSaveURL() const;
```

Comment:
getWhiteboardDefaultSaveURL() -> getWhiteboardSaveFolder ?

Please add unit tests for this new functionality in tests/cpp/core/ConfigurationTests.cpp.